### PR TITLE
redesign food modal

### PR DESCRIPTION
**Pull request by minimatime**
## PR Title

Issue Number(s): #131 

What does this PR change and why?

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria is met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [ ] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: src/services/pets.ts [hunk 1]
```diff
@@ -45,8 +45,8 @@ export default class PetService {
       petType,
       xpGained: 0,
       xpLevel: 0,
-      coins: 0,
-      food: 0,
+      coins: 1000,
+      food: 5,
```
[the remark this answers is not in the document]
Comment (minimatime) [reply src/services/pets.ts]: yes this was from two weeks ago when i couldn't really access anything in the feed because i was broke and hungry

File: src/pages/index.tsx [hunk 2]
```diff
@@ -109,7 +131,7 @@ export default function Home({
             <NavButton buttonType="store" />
             <NavButton buttonType="bag" />
             <NavButton buttonType="log" />
-            {pet.food > 0 && <FeedButton />}
+            {pet.food >= 0 && <FeedButton />}
```
[the remark this answers is not in the document]
Comment (minimatime) [reply src/pages/index.tsx]: again was doing this for testing oops

File: src/components/modals/FoodModal.tsx [hunk 3]
```diff
@@ -35,41 +54,59 @@ export default function FoodModal({ foods }: { foods: string[] }) {
...
+              <Image
+                src={`${baseImageUrl}/${clickedFood}.svg`}
+                alt={clickedFood}
+                width={40}
+                height={40}
+                className="object-contain"
+              />
+              {/* <span className="text-black text-xl font-bold">
+                Food: {getFoodCount(clickedFood)} Left
+              </span> */}
```
Comment (rgodha24) [inline src/components/modals/FoodModal.tsx]: This is kinda unclear in the design, but I think this should show the 'total food' that the user has saved up (which is on the pet table in the db). I think we talked about changing the icon away from the pizza to make this more clear.

But basically what im saying is you dont really need to pass in the clickedFood into this, because the pet has a certain number of _food_, not a certain number of pizza, and a different number of avocados.
